incubator/hnc/pkg/kubectl/root.go

```diff
@@ -20,6 +20,7 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/kubernetes-sigs/multi-tenancy/incubator/hnc/pkg/reconcilers"
```

Comment:
Can you move this to the next "paragraph" on line 34? We've tried to keep the imports in three sections:

* Standard library
* External dependencies
* Internal imports

Thanks!